pkg/apis/crd.projectcalico.org/v1/felixconfig.go

```diff
@@ -349,6 +357,11 @@ type FelixConfigurationSpec struct {
 	// Set source-destination-check on AWS EC2 instances. Accepted value must be one of "DoNothing", "Enabled" or "Disabled".
 	// [Default: DoNothing]
 	AWSSrcDstCheck *AWSSrcDstCheckOption `json:"awsSrcDstCheck,omitempty" validate:"omitempty,oneof=DoNothing Enable Disable"`
+
+	// TPROXYMode sets whether traffic is directed through a transparent proxy for further processing
+	// or not and how is the proxying done.
```

Comment:
I think since the options are just "Enabled" and "Disabled" we can probably remove the portion explaining that this tells how the proxying is done since this just decides whether we set up the tproxy to go through envoy for select traffic.

Reply:
fixed in https://github.com/tigera/operator/pull/1505/commits/c5ff8851d37c8e53234c9eb3276c45b555673fbd